resources/artifact-ignores.properties

```diff
@@ -732,6 +732,16 @@ sinatra-chef-builder = https://www.jenkins.io/security/plugins/#suspensions
 
 # The plugin has been closed source since 2015
 starteam = https://github.com/jenkins-infra/update-center2/pull/571
+# Closed source/end of life since 2018
+openscada-jenkins-exporter = https://github.com/jenkins-infra/update-center2/pull/682
```

Comment:
It's not closed source just because its latest source hoster is offline.

The latest release has a source jar: https://repo.jenkins-ci.org/artifactory/releases/org/openscada/jenkins/openscada-jenkins-exporter/0.0.2/openscada-jenkins-exporter-0.0.2-sources.jar